tools-local/tasks/mobile.tasks/WasmAppBuilder/WasmAppBuilder.cs

```diff
@@ -25,10 +25,6 @@ public class WasmAppBuilder : Task
     [Required]
     public string? MainJS { get; set; }
 
-    // If true, continue when a referenced assembly cannot be found.
-    // If false, throw an exception.
-    public bool SkipMissingAssemblies { get; set; }
```

Comment:
So the thing that is going to have to change in the runtime test is where it uses this parameter: https://github.com/dotnet/runtime/blob/4730697ad2c218f68dad1f5e788a897ac4fb01ea/src/tests/Common/wasm-test-runner/WasmTestRunner.proj#L35

Assuming this still gets all the assemblies and puts them in the right place, I don't think other changes are required for the wasm tests.

Reply:
I added a task that preserves what you need for the runtime tests.  That way we can remove using MDLC from WasmAppBuilder.

Reply:
Ok, looks good to me as far as the runtime tests go.